### PR TITLE
Update elabapy for improved eLabFTW API (get_all_items) 

### PR DESCRIPTION
**Pull request by JAC28**
### Problem ### 
API was updated but package was not (see issue #17 
### Changes ###
added additional arguments in get_all_items method and changed params_name to 'params' instead of default 'data

### CODE REVIEW

**Review by NicolasCARPi — CHANGES_REQUESTED** [review]
Thank you for this. Shouldn't this be also the same for `get_all_experiments()`? Also, this change will break existing code, so params should have a default value to be backward compatible.

As for the param_name change, it needs to be `data` or `files` for the requests library. This should be made clear in a comment.

**Comment by JAC28** [comment]
Hi @NicolasCARPi, you've mententioned two valid points thus I've added the functionality for experiments as well and set default values. 
With param_name I am confused since it only works for me with `params`. This is also in accordance with the [doku](https://docs.python-requests.org/en/latest/api/#requests.get).

**Comment by NicolasCARPi** [comment]
Yes you're correct for the `params` stuff (I didn't run your code yet). In fact it took me a few minutes to remember why I put this here... You can add a comment linking to the request doc in the function comment so it is clear for future devs, including us ;)

File: elabapy/Manager.py [hunk 1]
```diff
@@ -24,23 +24,23 @@ def create_template(self):
         """
         return self.send_req("templates", verb='POST')
 
-    def get_all_experiments(self):
+    def get_all_experiments(self, params={'limit':15,'offset':0}):
         """
             This function returns a list of all experiments.
         """
-        return self.send_req("experiments/")
+        return self.send_req("experiments/", params, param_name='params')
```
Comment (JAC28) [inline elabapy/Manager.py:31]: param_name has to be 'params' to be in accordance to the requests [doku](https://2.python-requests.org/en/master/api/#requests.get)

File: elabapy/Manager.py [hunk 2]
```diff
@@ -27,6 +27,8 @@ def create_template(self):
     def get_all_experiments(self, params={'limit':15,'offset':0}):
         """
             This function returns a list of all experiments.
+            params is a dict with keys 'limit' and/or 'offset'
+            param_name has to be 'params' to be in accordance with the current requests-doku (https://2.python-requests.org/en/master/api/#requests.get)
```
Comment (NicolasCARPi) [inline elabapy/Manager.py:31]: use: "current requests documentation"

Comment (JAC28) [reply elabapy/Manager.py]: I've added your suggestion within the last commit.

**Review by NicolasCARPi — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**